frontend/src/arkham/components/ChaosBag.vue

```diff
@@ -154,13 +154,49 @@ const choose = (idx: number) => emit('choose', idx)
   align-items: center;
   justify-content: center;
   flex-wrap: wrap;
+   @media (max-width: 800px) and (orientation: portrait) {
+    position:absolute;
+    width: 100%;
+    left: 0;
+   }
 }
 
 .token-preview {
   display: flex;
   flex-wrap: wrap;
   flex-direction: row;
   justify-content: center;
+  @media (max-width: 800px) and (orientation: portrait) {
+    display: grid;
+    grid-template-columns: 0.11fr 0.11fr 0.11fr 0.33fr 0.11fr 0.11fr 0.11fr;
```

Comment:
I think this can just be 
```suggestion
    grid-template-columns: 1fr 1fr 1fr 3fr 1fr 1fr 1fr;
```
Which should be similar